addons/medical/stringtable.xml

```diff
@@ -5623,5 +5623,23 @@
             <Chinesesimp>关闭盖子</Chinesesimp>
             <Korean>뚜껑 닫기</Korean>
         </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_allowDeadBodyMovement_DisplayName">
+            <English>allowDeadBodyMovement</English>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_allowDeadBodyMovement_Description">
+            <English>Not referenced anywhere in the source, does nothing ?!</English>
+        </Key>
+        <Key ID="STR_ACE_Medical_MedicalSettings_allowUnconsciousAnimationOnTreatment_DisplayName">
+            <English>Unconscious animation during treatment</English>
```

Comment:
That name looks like it would be too long to be correctly displayed in the settings menu.

Reply:
Used https://github.com/acemod/ACE3/blob/master/addons/medical/stringtable.xml#L5593-L5594 as a reference.

Reply:
Tested? I made the names pretty long and this could still fit iirc.

Reply:
*Devbuild result*
![ace_medical](https://user-images.githubusercontent.com/20744006/36043848-fe8271f4-0dd0-11e8-910e-a70ed602bd35.jpg)

Reply:
FYI if it fits for one resolution / aspect ratio / ui scale, it fits for all.